### DISCUSSION TITLE
Accessing Yeti's HTML files

### DISCUSSION THREAD

**Discussion by bfeddes** [YetiForceCompany/YetiForceCRM#17071] [How to]
Just a quick trivial question. Where would I go to access Yeti's HTML files, for example if I wanted to edit how a certain module like Tickets, the Dashboard, etc look? What is the file path for various HTML files?

Thanks in advance!

**Comment by mariuszkrzaczkowski** [comment]
https://github.com/YetiForceCompany/YetiForceCRM/tree/developer/layouts/basic/modules
